docs/en/ingest-management/release-notes/release-notes-8.6.asciidoc

```diff
@@ -75,6 +75,28 @@ To work around this problem, uninstall the {agent} and install it again with
 {fleet-server} enabled during the bootstrap process.
 ====
 
+[[known-issue-2103-8-6-2]]
+.Installing {agent} on MacOS Ventura fails with a permissions error if Full Disk Access is not granted to the application used for installation.
```

Comment:
```suggestion
.Installing {agent} on MacOS Ventura fails with a permissions error if Full Disk Access is not granted to the application used for installation
```

Reply:
If this is valid in 8.6.2, it should be added to 8.6.1, too. 

Incidentally I was able to install 8.6.1 successfully on Ventura so I thought this issue was resolved, but I guess it's still a problem? Maybe I fixed the issue myself without remembering. :-)

Reply:
It is conditional on a few different things, including the version of MacOS Ventura. I should probably reword this to "can fail" and not "fails".